internal/runbits/errors/errors.go

```diff
@@ -197,7 +215,7 @@ func ReportError(err error, cmd *captain.Command, an analytics.Dispatcher) {
 	var action string
 	errorMsg := err.Error()
 	if IsReportableError(err) {
-		multilog.Critical("Returning error:\n%s\nCreated at:\n%s", errs.JoinMessage(err), stack)
+		multilog.Critical("Returning error (ID: %s):\n%s\nCreated at:\n%s", getErrorId(), errs.JoinMessage(err), stack)
```

Comment:
This will call `rollbar.Critical`, which will forward our message and error ID to rollbar.